habitat-lab/habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -1026,7 +1025,7 @@ def settle_sim(
                                 mn.Color4.red(),
                             )
                         ],
-                    )
+                    ).save(self.dbv.output_path, prefix="unstable_")
```

Comment:
Could you add a small comment here explaining what this file is?
I would imagine people `CTRL+F`ing the codebase to figure out where `unstable_` files comes from.